debezium-core/src/main/java/io/debezium/relational/RelationalSnapshotChangeEventSource.java

```diff
@@ -117,6 +132,11 @@ public SnapshotResult<O> doExecute(ChangeEventSourceContext context, O previousO
             LOGGER.info("Snapshot step 5 - Reading structure of captured tables");
             readTableStructure(context, ctx, previousOffset);
 
+            if (snapshottingTask.snapshotData()) {
+                LOGGER.info("Snapshot step 5.a - Creating connection pool");
```

Comment:
Please replace ardodced numbers in the log messages with somehing like `step {}` and have a variable stepCounter, that will be passed to the log message and incremented so step numbers are dynamically assigned and there is no need for suffixes.